### PR TITLE
Tests/models

### PR DESCRIPTION
**Pull request by ocampd1**

### CODE REVIEW

File: chucks-pick3-mobile/src/app/models/pick3-draw-date-card.domain.spec.ts [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+import {Pick3DrawTimeEnum} from "./pick3-draw-time.enum";
+import {Pick3LotteryService} from "../services/pick3-lottery.service";
+import {IonicToastNotificationService} from "../services/ionic-toast-notification.service";
+describe('Pick3DrawDateCardDomain', () => {
+
+  beforeEach(() => {
+
+  });
+  let model = new Pick3DrawDateCardDomain({
+    drawDate : new Date(),
```
Comment (pouncilt) [inline chucks-pick3-mobile/src/app/models/pick3-draw-date-card.domain.spec.ts:13]: Create a const for date before using the date here.  And then later you can expect for the exact date.

File: chucks-pick3-mobile/src/app/models/pick3-draw-date-card.domain.spec.ts [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    winningNumberDigits: [4,6,2],
+    drawDateIcon: new Date(),
+    slideNumber: 7
+
+  });
+
+  // drawDate
+  it('should have a drawDate defined', function () {
+    const drawDate = model.getDrawDate();
+    expect(drawDate).toBeDefined();
```
Comment (pouncilt) [inline chucks-pick3-mobile/src/app/models/pick3-draw-date-card.domain.spec.ts:30]: In addition to this, you an expect for exact date previous created.  See comments above.

**Review by pouncilt — APPROVED** [review]
Looking good!

File: chucks-pick3-mobile/src/app/models/pick3-draw-date-card.domain.spec.ts (hunk 2 above)
Comment (ocampd1) [reply chucks-pick3-mobile/src/app/models/pick3-draw-date-card.domain.spec.ts]: Renamed and inserted use for date

File: chucks-pick3-mobile/src/app/models/pick3-draw-date-card.domain.spec.ts (hunk 1 above)
Comment (ocampd1) [reply chucks-pick3-mobile/src/app/models/pick3-draw-date-card.domain.spec.ts]: added date and replaced expected

**Review by ocampd1 — COMMENTED** [review]
Changed service on enum to enumValue

**Pull request merged**